NotesApp/Scenes/Editor/EditorViewController.swift

```diff
@@ -11,27 +11,37 @@ protocol AddNoteDelegate: AnyObject {
     func didAddNote()
 }
 
-class EditorViewController: UIViewController {
+final class EditorViewController: UIViewController {
     // MARK: - Properties
     private let viewModel: EditorViewModel
     weak var delegate: AddNoteDelegate?
 
-    private lazy var customBackButton = CustomBackButton(
-        iconName: "chevron.left",
-        pointSize: 20,
-        backgroundColor: .darkGray,
-        tintColor: .white,
-        cornerRadius: 14,
-        size: 48
-    )
+    // MARK: - Components
+    private lazy var customBackButton: UIButton = {
+        let button = CustomBackButton(
+            iconName: "chevron.left",
+            pointSize: 20,
+            backgroundColor: .darkGray,
+            tintColor: .white,
+            cornerRadius: 14,
+            size: 48
+        )
+        
```

Comment:
Left some numbers here, move to constraints, you can create extension inside this file since you have few numbers here